packages/pages/scripts/install_and_test.js

```diff
@@ -0,0 +1,24 @@
+import path from "path";
+import fs from "fs";
+import * as child_process from "child_process";
+
+// Gets the path to the packaged version of PagesJS
+const latestPath = path.resolve("latest");
+const packagePath = path.resolve(
+  "latest",
```

Comment:
What is latest?

Reply:
It's the directory that pages gets packed into